spec.go

```diff
@@ -6,7 +6,7 @@ const (
 	tillerDefaultNamespace = "kube-system"
 	tillerImageSpec        = "gcr.io/kubernetes-helm/tiller:v2.8.2"
 	tillerLabelSelector    = "app=helm,name=tiller"
-	tillerPodName          = "tiller"
+	tillerPodName          = "tiller-giantswarm"
```

Comment:
I wondered whether the pod name should include the configured namespace. But this makes sense since its our helmclient library.

Reply:
Exactly, it's in fact not related to the namespace in this case, but we could easily bind the two together if needed.